frontend/ui/uimanager.lua

```diff
@@ -237,10 +237,11 @@ function UIManager:close(widget, refreshtype, refreshregion, refreshdither)
     end
 end
 
--- Schedule an execution task; task queue is in ascending order
+-- Schedule an execution task; task queue is in descending order
 function UIManager:schedule(sched_time, action, ...)
     local lo, hi = 1, #self._task_queue
-    -- Rightmost binary insertion
+    self._task_queue.len = hi + 1
```

Comment:
I'd go w/ `n` to follow Lua traditions ;).

(And set it to 0 at declaration time).

Reply:
You mean something like `self._task_queue = {n = 0}`?

Reply:
Yup!

Reply:
Or, well, not  `self.`, as I mean the actual declaration in @ ~L39